src/components/shared/Draggable.js

```diff
@@ -13,18 +30,29 @@ import React, { PureComponent, PropTypes } from 'react';
  * During the drag, the additional className 'dragging' is set on the element.
  */
 export default class Draggable extends PureComponent {
-  constructor(props) {
+  props: Props;
+  state: {
+    dragging: boolean,
+  };
+  _container: ?HTMLDivElement;
```

Comment:
Usually, in our code, we try to be stricter when we can; in this case _handlers can be either null or the object. So here `container` is `HTMLDivElement | null`.